core/node/eth_sender/src/eth_tx_manager.rs

```diff
@@ -552,11 +553,21 @@ impl EthTxManager {
             let l1_block_numbers = self
                 .l1_interface
                 .get_l1_block_numbers(operator_to_track)
-                .await?;
-            METRICS.track_block_numbers(&l1_block_numbers);
+                .await;
+
+            if let Err(ref error) = l1_block_numbers {
+                // Web3 API request failures can cause this,
+                // and anything more important is already properly reported.
+                tracing::warn!("eth_sender error {:?}", error);
+                if error.is_retriable() {
+                    METRICS.l1_transient_errors.inc();
+                    continue;
```

Comment:
I'm not sure if it is a good idea to continue eth sender if SL's API is down. I think it's worth panicking if the loop doesn't make any progress for some time.

Reply:
We don't want to shut down the server if the sl api is down. 
especially given that our reliability is lower than eth

Reply:
Well, this way we'll see that eth sender is in error loop in 1 hour (default no-batch-commited alert interval). I think we need some compromise here and panicking/alerting if there are no successful loop iterations for 5-10 minutes is a good option IMO. This applies only for eth sender, I don't suggest doing this for gas-adjuster>state-keeper

Reply:
can we use l1_transient_errors ? for it

Reply:
I don't think so, at least not in a reliable manner. Anyway, looks like it has a bigger scope so let's maybe address it in a separate PR